db/models/assessment_record/assessment_records.py

```diff
@@ -0,0 +1,82 @@
+from sqlalchemy import Index
+from db import db
+from db.models.assessment_record.enums import Status
+from sqlalchemy import cast
+from sqlalchemy import Computed
+from sqlalchemy import func
+from sqlalchemy.dialects.postgresql import ENUM
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.dialects.postgresql import TEXT
+from sqlalchemy.dialects.postgresql import UUID
+
+
+class AssessmentRecords(db.Model):
+    __tablename__ = "assessment_records"
+
+    application_id = db.Column(
+        "application_id", UUID(), primary_key=True
+    )
+
+    short_id = db.Column("short_id", db.Text(), nullable=False)
+
+    type_of_application = db.Column(
+        "type_of_application", db.Text(), index=True, nullable=False
+    )
+
+    project_name = db.Column(
+        "project_name", db.Text(), index=True, nullable=False
+    )
+
+    funding_amount_requested = db.Column(
+        "funding_amount_requested", db.Float(), index=True, nullable=False
+    )
+
+    round_id = db.Column("round_id", UUID(), index=True, nullable=False)
+
+    fund_id = db.Column("fund_id", UUID(), index=True, nullable=False)
+
+    langauge = db.Column("langauge", db.Text(), default="en", nullable=False)
```

Comment:
Can we make this a String type rather than Text? (Or an ENUM with 'cy' and 'en' as values?